src/test/java/teammates/sqllogic/core/AccountsLogicTest.java

```diff
@@ -56,19 +57,22 @@ public void setUpMethod() {
     }
 
     @Test
-    public void testDeleteAccount_accountExists_success() {
+    public void testDeleteAccount_accountExists_success()
+            throws InvalidParametersException, EntityAlreadyExistsException {
         Account account = generateTypicalAccount();
         String googleId = account.getGoogleId();
 
-        when(accountsLogic.getAccountForGoogleId(googleId)).thenReturn(account);
+        when(accountsLogic.createAccount(account)).thenReturn(account);
```

Comment:
Why does a unit test for deleteAccount require createAccount to be stubbed?

Reply:
Oops! My mistake, sorry about that!!